tests/Feature/Http/Controllers/AnnounceControllerTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\Feature\Http\Controllers;
+
+use App\Helpers\Bencode;
+use App\Models\Torrent;
+use App\Models\User;
+use GroupsTableSeeder;
+use Tests\TestCase;
+
+/**
+ * @see \App\Http\Controllers\AnnounceController
+ */
+class AnnounceControllerTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function announce_returns_an_ok_response()
+    {
+        $this->seed(GroupsTableSeeder::class);
+
+        $user = factory(User::class)->create([
+            'can_download' => true,
+        ]);
+
+        factory(Torrent::class)->create([
+            'info_hash' => bin2hex(md5('foo')),
+            'status'    => 1, // Approved
+        ]);
+
+        $response = $this->get(route('announce', [
+            'passkey'    => $user->passkey,
+            'info_hash'  => md5('foo'),
+            'peer_id'    => 'bar',
```

Comment:
Should be 20-byte string used as a unique ID for the client, generated by the client at startup.

Reply:
The actual value is inconsequential unless we assert against it, but it can't hurt to use a realistic value if for no other reason than documenting it (in code) for use in a future test that actually checks the value.